Libs/MRML/Core/vtkMRMLMeasurement.cxx

```diff
@@ -337,3 +335,9 @@ vtkMRMLNode* vtkMRMLMeasurement::GetInputMRMLNode()
 
   return nullptr;
 }
+
+//----------------------------------------------------------------------------
+void vtkMRMLMeasurement::Compute()
```

Comment:
Does this compile? you defined `virtual void Compute() = 0;` in the header.

Reply:
An oversight. I'll remove the body. Sorry!